lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,144 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid $gray-l3;
```

Comment:
NIT: As general feedback, properties should be ordered using the Concentric CSS ordering list - reference included below: 
https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style.css 

In this case for example, display --> border --> padding --> width.